### ISSUE TITLE
Addition of farsi language

### ISSUE THREAD

**Issue by mahdiasli** [zhw2590582/ArtPlayer#389]
HI ! @zhw2590582 
Artplayer is so graet !!!
 but it´s necesary to add farsi to languages.
please add it over than 1000 programer are waiting for it
Thank you !
[fa-lan.txt](https://github.com/zhw2590582/ArtPlayer/files/9702935/fa-lan.txt)

**Comment by zhw2590582** [comment]
Thanks for your PR, I have merged your PR and will publish it in the next version soon

**Comment by mahdiasli** [comment]
how can i use it now cuse i need it :((

**Comment by zhw2590582** [comment]
I just packaged an unreleased version `4.5.8`, you can download this script directly:

https://raw.githubusercontent.com/zhw2590582/ArtPlayer/master/packages/artplayer/dist/artplayer.js

```js
var art = new Artplayer({
    container: '.artplayer-app',
    url: '/assets/sample/video.mp4',
    lang: 'fa',
});
```

**Comment by mahdiasli** [comment]
thank you it was big help ! 
please relase the new version in npm more of us use artplayer npm package as dependency and impatiently wating for release

thank you with all heart

**Comment by zhw2590582** [comment]
yes i will release it in a few days

**Comment by zhw2590582** [comment]
The latest version has been released: `artplayer@4.5.8`
